contracts/controller/Controller.sol

```diff
@@ -188,11 +180,10 @@ contract Controller is ControllerInterface {
    * @param _permissions the permissions the new scheme will have
    * @return bool which represents a success
    */
-    function registerScheme(address _scheme, bytes32 _paramsHash, bytes4 _permissions, address _avatar)
```

Comment:
Should remove `_paramsHash` from the Controller entirely.

Reply:
yea. though not totally sure re that ...as if someone would like to have it own custom universal scheme..he could use this mechanism

Reply:
If we want to allow it we can leave it, but I don't think we want to allow that...